### PR TITLE
Siddev

### PR DESCRIPTION
**Pull request by Siddarthapogula**

### CODE REVIEW

File: packages/alea-frontend/pages/api/access-control/user-members/[aclid].ts
Comment (abchugh) [inline packages/alea-frontend/pages/api/access-control/user-members/[aclid].ts:1]: where is this used? remove if unused

File: packages/alea-frontend/pages/acl/new.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  const handleSubmit = async () => {
+    const newAcl: CreateACLRequest = {
+      id: aclId,
+      description,
+      memberUserIds,
+      memberACLIds,
+      updaterACLId,
+      isOpen,
+    };
+    console.log('New ACL:', newAcl);
```
Comment (abchugh) [inline packages/alea-frontend/pages/acl/new.tsx:66]: remove unnecessary console.log everywhere

File: packages/alea-frontend/pages/acl/new.tsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            variant="outlined"
+            size="small"
+            value={tempMemberUserId}
+            onChange={(e) => setTempMemberUserId(e.target.value)}
+            onKeyDown={handleAddMemberId}
+            sx={{ mb: '10px' }}
+            fullWidth
+            InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">
```
Comment (abchugh) [inline packages/alea-frontend/pages/acl/new.tsx:119]: does this need `cursor: pointer` css property?

**Review by abchugh — APPROVED** [review]

File: packages/alea-frontend/pages/acl/new.tsx (hunk 2 above)
Comment (Siddarthapogula) [reply packages/alea-frontend/pages/acl/new.tsx:119]: we don't need to add it.

**Pull request merged**